README.md

```diff
@@ -1,2 +1,17 @@
-# shlifim
-Shlifim - Beyond 05, team 3 project
+# Shlifim Project - beyond 05 team 3
+
+![shlifim_logo_2](https://user-images.githubusercontent.com/40122521/110316976-2990f880-8014-11eb-8ee8-96c21dd4b29f.png)
+
+Shlifim is an open source forum based web application for helping students find answers
+to their homework and assignments and also share summaries.
+
+The develpoment of this project is in [Vagrant](https://www.vagrantup.com/) environment using [PipEnv](https://github.com/pypa/pipenv), [Django](https://www.djangoproject.com/) and [SQLite](https://www.sqlite.org/index.html) as DB
```

Comment:
Parts of this statement are not yet true, perhaps only leave the true parts?

Also I suggest including information for developers about how to get started with contributing to this project.